### PR TITLE
Address flakiness with i18n integration tests.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: server/integration_tests/nl_test.py [hunk 1]
```diff
@@ -131,6 +138,33 @@ def handle_response(self,
           self.assertEqual(dbg["main_place_dcid"], expected["main_place_dcid"])
           self.assertEqual(dbg["main_place_name"], expected["main_place_name"])
 
+  def handle_i18n_response(self, resp, i18n_lang):
```
Comment (pradh) [inline server/integration_tests/nl_test.py:141]: Should we put this in a common place?

**Review by pradh — APPROVED** [review]
Thanks for the change!

File: server/integration_tests/nl_test.py (hunk 1 above)
Comment (keyurva) [reply server/integration_tests/nl_test.py:141]: For tests, prefer to keep assertion logic in the same file. I think `handle_response` in both files is also similar but kept separate for similar reasons?

Comment (pradh) [reply server/integration_tests/nl_test.py:141]: Yeah, that's also worth deduping.  We can have a TODO

Comment (pradh) [reply server/integration_tests/nl_test.py:141]: Sure

**Pull request merged**